test/util/PolarUtils.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { RADIAN, degreeToRadian, getMaxRadius, polarToCartesian, radianToDegree } from '../../src/util/PolarUtils';
```

Comment:
To reduce the blast radius of this PR I suggest too add all tests separately. Type chances can cause pain for customers, tests never do. 

Thus I would be happy to get the tests merged in first, while paying more attention to the changes of types in the review.